Ochowo/iReporter
# chore(server):set up database

**What does this PR do?**
Sets up a Postgres database

**Description of the task to be completed**
Ensure the following tables are created
`users table`
`redflags table`
`interventions table`

**How should this be tested manually**
After you clone the repo, cd into it and run `npm install` to install node modules. Make sure you have `nodejs` and `express` installed. Afterwards, `run npm start:dev to initiate the app and tables will be created.

What are the relevant pivotal stories
N/A
